public/src/css/includes/index.less

```diff
@@ -6,6 +6,10 @@
 
 .model-actions, .options {
 
+  .form-control {
+    height: 42px;
```

Comment:
I'm not too concerned with this since developers don't currently have access to the less files and we are planning on moving to scss (or postcss?) anyway, but this should go in some variable.

Reply:
Yeah, it needs a tidy up. I'll leave it as is for now, because there is no real convention for our CSS yet :(